src/main/java/nl/tudelft/contextproject/audio/BackgroundMusic.java

```diff
@@ -191,15 +178,11 @@ public synchronized AudioNode getCurrent() {
 	
 	/**
 	 * Called to indicate an update.
-	 * 
-	 * @param tpf
-	 * 		tick indicator
 	 */
-	public void update(double tpf) {
+	public void update() {
 		AudioNode an = current;
 		if (an == null) return;
-		
-		//Play the next song if the last one is done playing.
+
 		if (an.getStatus() == AudioSource.Status.Stopped) {
 			next();
 		}
```

Comment:
All comments can be re-added, but I was of the opinion they didn't add any extra understandability to the code. As I said, I'm fine with readding them.